twitter.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Page</title>
+    <script src="https://kit.fontawesome.com/59ab1ef207.js" crossorigin="anonymous"></script>
+    <link rel = "stylesheet" href ="style.css" />
+</head>
+<body>
+    <div class="container">
+        <div class="left-menu"> 
+            <div class="logo">
+                <img src = "https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/Twitter-logo.svg/512px-Twitter-logo.svg.png?20220821125553"  alt = "logo"/>
+            </div>
+            <div class="icons">
```

Comment:
The only possible improvement I would suggest, is putting the anchor tags in a nav tag for accessibility. Either way - fantastic job bud!

Reply:
That's a really good point.  i totally forgot to differentiate the sections in html.